python/pyraphtory/pyraphtory/fileutils.py

```diff
@@ -0,0 +1,77 @@
+import requests
+from requests.adapters import HTTPAdapter, Retry
+import shutil
+import os.path
+import hashlib
+
+_URL = 'url'
+_SHA256 = 'sha256'
+
+RAPHTORY_JARS_HASH = {
+    '0.2.0a0': {
+        _URL: 'https://github.com/Raphtory/Raphtory/releases/download/v0.2.0a0/core-0.2.0a0.jar',
```

Comment:
Have a look at https://stackoverflow.com/questions/17341122/link-and-execute-external-javascript-file-hosted-on-github

it shows a better way to link github files via a CDN